packages/@react-spectrum/menu/test/MenuTrigger.test.js

```diff
@@ -1225,4 +1230,51 @@ describe('MenuTrigger', function () {
       });
     });
   });
+
+  describe('portalContainer', () => {
+    function InfoMenu(props) {
+      return (
+        <Provider theme={theme}>
+          <MenuTrigger UNSTABLE_portalContainer={props.container}>
+            <ActionButton aria-label="trigger" />
+            <Menu>
+              <Item key="1">One</Item>
+              <Item key="">Two</Item>
+              <Item key="3">Three</Item>
+            </Menu>
+          </MenuTrigger>
+        </Provider>
+      );
+    }
+    function App() {
+      let [container, setContainer] = React.useState();
+      return (
+        <>
+          <InfoMenu container={container} />
+          <div ref={setContainer} data-testid="custom-container" />
+        </>
+      );
+    }
+    it('should render the menu in the portal container', async () => {
+      let {getByRole, getByTestId} = render(
+        <App />
+      );
+
+      let button = getByRole('button');
+      await user.click(button);
+
+      expect(getByRole('menu').closest('[data-testid="custom-container"]')).toBe(getByTestId('custom-container'));
+    });
+    it('should render the menu tray in the portal container', async () => {
+      windowSpy.mockImplementation(() => 700);
```

Comment:
Seeing this test made me realize we didn't have a test for Menu loading as a tray on mobile. Submenu does have one. :)